### DISCUSSION TITLE
How to stop showing errors

### DISCUSSION THREAD

**Discussion by oasissan** [ajaxorg/ace#5311] [Q&A] [answered]
I am trying to prevent the ide from showing errors I could not find anything in the wiki.

**Comment by whazor** [comment]
Hi, Ace is a code editor that you can embed into your own website and not an IDE. Do you have a screenshot or maybe the errors themselves?

**Reply by oasissan** [reply]
![image](https://github.com/ajaxorg/ace/assets/59906580/5c294470-df9e-4337-9fb0-e57964d64a2c)

I mean the syntax errors, etc.

**Reply by nightwing** [reply, marked as answer]
If you want to disable all workers use `editor.setOption("useWorker", false)`

**Reply by oasissan** [reply]
Thank you

**Comment by phiden1** [comment]
Code '' useworld....put3780d2f7 ''
